docs/IntegrationWithExistingApps.md

```diff
@@ -720,6 +720,8 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
 
 That's it, your activity is ready to run some JavaScript code.
 
+> If your app is targeting the Android `api level 23` or greater, make sure you have, for the development build, the `overlay permission` enabled. You can check it with `Settings.canDrawOverlays(this);`. This is required because, if your app produces an error in the react native component, the error view is displayed above all the other windows. Due to the new permissions system, introduced in the api level 23, the user needs to approve it.
```

Comment:
@satya164 Is there any plan to support the newer `targetSdkVersion` any time soon? I recently became a victim of setting `targetSdkVersion` to 25 after updating the Android appcompat library to 25.0.0 and started to see the above described problem. @leeight was kind enough to let me know that the `targetSdkVersion` should be set to 22 or things may not work and directed me to this PR as well. But after I changed the `targetSdkVersion` to 22, I stopped seeing the permissions dialog boxes when a particular resource is accessed for the first time that started to appear when I change the `targetSdkVersion` to  25. The Android app users may start to get used to that UX as time go by and expect the same behavior from other apps as well. I think it will also definitely help react-native users if it is announced in the doc that the `targetSdkVersion` should be set to 22 and not higher than that. Right now, the doc just says that API level 16 and above are supported and nothing about the `targetSdkVersion`.